src/appengine/libs/issue_management/jira/__init__.py

```diff
@@ -172,6 +180,10 @@ def issue_url(self, issue_id):
     url = config.jira_url + '/browse/' + str(issue_id)
     return url
 
+  def find_issues_url(self, keywords=None, only_open=None):  # pylint: disable=unused-argument
```

Comment:
Add a "FIXME: Add support for keywords and only_open arguments" at end of function.

Reply:
Added, please check.